File: SoftUni Lessons/Python Development/Python Basics April 2019/Lessons and Problems/11.1 - For Loop More Exercises/07. Football League.py
stadium_capacity = int(input())
count_fens = int(input())

a = 0
b = 0
v = 0
g = 0

for i in range(count_fens):
    sector = input()
    if sector == "A":
        a += 1
    if sector == "B":
        b += 1
    if sector == "V":
        v += 1
    if sector == "G":
        g += 1

print(f"{(a / count_fens * 100):.2f}%")
print(f"{(b / count_fens * 100):.2f}%")
print(f"{(v / count_fens * 100):.2f}%")
print(f"{(g / count_fens * 100):.2f}%")
print(f"{(count_fens / stadium_capacity * 100):.2f}%")

File: SoftUni Lessons/JavaScript Development/JavaScript Basics March 2020/12. УПРАЖНЕНИЕ ЦИКЛИ/For-Loop - More Exercises/07. Football League.js
function footballLeague(input) {
    // Mask
    let stadium_capacity = Number(input.shift());
    let count_fens = Number(input.shift());

    let a = 0;
    let b = 0;
    let v = 0;
    let g = 0;

    for (let i = 1; i <= count_fens; i++) {
        let sector = input.shift();
        
        if (sector == "A") {a += 1;}
        if (sector == "B") {b += 1;}
        if (sector == "V") {v += 1;}
        if (sector == "G") {g += 1;}
    }

    console.log(`${(a / count_fens * 100).toFixed(2)}%`);
    console.log(`${(b / count_fens * 100).toFixed(2)}%`);
    console.log(`${(v / count_fens * 100).toFixed(2)}%`);
    console.log(`${(g / count_fens * 100).toFixed(2)}%`);
    console.log(`${(count_fens / stadium_capacity * 100).toFixed(2)}%`);
}

footballLeague([1000, 12, "A", "A", "V", "V", "A", "G", "A", "A", "V", "G", "V", "A"]);  
// Should return:

// 50.00%
// 0.00%
// 33.33%
// 16.67%
// 1.20%
